dub.selections.json

```diff
@@ -4,9 +4,10 @@
 		"libevent": "2.0.1+2.0.16",
 		"libev": "5.0.0+4.04",
 		"openssl": "1.1.4+1.0.1g",
+		"libdparse": "0.2.1",
```

Comment:
Whoops, it's too easy to overlook this.
I should fix dub to at least always output a stable order.